trademapper/js/trademapper.js

```diff
@@ -93,7 +94,7 @@ define([
 	"text!../fragments/reopencustomcsv.html",
 	"text!../fragments/svgstyles.css",
 ],
-function($, d3, arrows, csv, filterform, mapper, route, yearslider,
+function($, d3, analytics, arrows, csv, filterform, mapper, route, yearslider,
```

Comment:
The argument order is wrong here: analytics should come after arrows. I noticed this because I was getting error messages in the console from trademapper.arrows.

Reply:
Argh yes, and it worked because it also calls arrows.init. The arguments are just ignored.